content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
 
-If you were to add percentile aggregations to your distribution metric (as shown in-app [Datadog Distribution Metric page][2]), the following timeseries would be additionally created:
-
-| Metric Name                              | Value | Datadog Metric Type |
-|------------------------------------------|-------|---------------------|
-| `p50:request.response_time.distribution` | `2 `  | GAUGE               |
+| Metric Name                              | Value | Datadog In-app Type |
+| ---------------------------------------- | ----- | ------------------- |
+| `p50:request.response_time.distribution` | `2`   | GAUGE               |
 | `p75:request.response_time.distribution` | `2`   | GAUGE               |
 | `p90:request.response_time.distribution` | `3`   | GAUGE               |
 | `p95:request.response_time.distribution` | `3`   | GAUGE               |
 | `p99:request.response_time.distribution` | `3`   | GAUGE               |
 
-**Note**: In the example above, the p50 (median) for `server:web_1` is `2` and the p50 for `server:web_2` is `1`. Agent-side aggregation would result in taking the median of these two median values, resulting in `1.5`. In reality, the global p50 (median) is the median of the combined set [1,1,1,1,1,2,2,2,2,3,3], which is `2`. This is the statistically accurate value that can be returned by a distribution metric's server-side aggregation.
+That is, for a distribution metric with added percentile aggregations during a given time interval, the following 10 aggregations are available: `count`, `sum`, `min`, `max`, `avg`, `p50`, `p75`, `p90`, `p95`, and `p99`.
 
-### Customization of tagging
+#### Customization of tagging
 
-This functionality allows you to control tagging for metrics where host-level granularity is not necessary. See the [Distribution Metric page][2] to learn more about whitelist-based tagging control. **Note**: The exclusion of tags with `!` is not accepted with this feature.
+This functionality allows you to control tagging for metrics where host-level granularity is not necessary. See the [Distribution Metric page][1] to learn more about whitelist-based tagging control.
 
-[1]: /developers/metrics/dogstatsd_metrics_submission/#distribution
-[2]: /metrics/distributions
+**Note**: The exclusion of tags with `!` is not accepted with this feature.
+
+
+[1]: https://app.datadoghq.com/metric/distribution_metrics
 {{% /tab %}}
-{{% tab "SET" %}}
+{{< /tabs >}}
+
+### Submission
+
+{{< tabs >}}
+{{% tab "COUNT" %}}
+
+Submit your COUNT type metrics from one of the following sources:
 
-**The `SET` metric type counts the number of unique occurrences of events over a period of time.**
+| Submission Source | Submission Method (python)           | Submission Type | Datadog In-App Type |
```

Comment:
Alphabetize this table
| ----------------- | ------------------------------------ | --------------- | ------------------- |
| [Agent check][3]  | `self.count(...)`                    | COUNT           | COUNT               |
| [Agent check][4]  | `self.monotonic_count(...)`          | COUNT           | COUNT               |
| [API][1]          | `api.Metric.send(type="count", ...)` | COUNT           | COUNT               |
| [DogStatsD][2]    | `dog.count(...)`                     | COUNT           | RATE                |
| [DogStatsD][2]    | `dog.increment(...)`                 | COUNT           | RATE                |
| [DogStatsD][2]    | `dog.decrement(...)`                 | COUNT           | RATE                |